openstack/containerinfra/v1/clustertemplates/requests.go

```diff
@@ -0,0 +1,66 @@
+package clustertemplates
+
+import (
+	"net/http"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+// CreateOptsBuilder Builder.
+type CreateOptsBuilder interface {
+	ToClusterCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts params
+type CreateOpts struct {
```

Comment:
Because of the number of fields, let's sort these in alphabetical order so they're easier to manage.

Reply:
I was following the order of the doc. Will change to alphabetical.